src/View/mainpage/DeleteROIWindow.py

```diff
@@ -7,6 +8,8 @@
 from src.Model import ROI
 import platform
```

Comment:
sort imports
(using isort)

Reply:
There is (in my opinion) a significant aesthetic need for consistency in imports across the board that isn't currently being met. In this class, for example, why are we importing QtWidgets _and_ 10 other classes from QtWidgets as well. In my opinion most files which depend on PyQt5 should import QtCore, QtGui, and QtWidgets as a baseline, and then call necessary classes from there. It's something I've been slowly trying to fix up as I work on files, but it does sometimes slip through the cracks. I think an entire branch should be dedicated to cleaning this up across the entire project.

Reply:
while I don't disagree with your statements,  my comment was more about having the import platform moved to the top section of imports (built-ins) and placed in alphabetical order within that section of imports.
the isort tool has a set of rules for ordering the imports so that they are easier to look through.

The rules regarding *how* to import, e.g. with respect to namespaces, etc. are important, and that's an issue worth raising, so I've done so.